changelog.d/15925.bugfix

```diff
@@ -0,0 +1 @@
+Fix Synapse starting with an empty `experimental_features` configuration setting.
```

Comment:
```suggestion
Fix Synapse failing to start with an empty `experimental_features` configuration setting.
```

Bonus points for "Fix a bug introduced in ..."

Reply:
```suggestion
Fix a bug introduced in 1.86.0 where Synapse starting with an empty `experimental_features` configuration setting.
```